packages/devextreme/js/ui/map.d.ts

```diff
@@ -205,6 +205,24 @@ export interface dxMapOptions extends WidgetOptions<dxMap> {
      * @public
      */
     focusStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default { mapId: '', useAdvancedMarkers: true }
+     * @public
+     */
+    providerConfig?: {
+      /**
+       * @docid
+       * @default ""
+       */
+      mapId?: string;
+      /**
+       * @docid
+       * @default true
+       * @deprecated
```

Comment:
Why you add property 'useAdvancedMarkers' and mark it as deprecated?

Reply:
Because we want it deprecated on release according to spec